idea/testData/inspectionsLocal/redundantCompanionReference/inEnumEntry.kt

```diff
@@ -0,0 +1,10 @@
+// PROBLEM: none
+// DISABLE-ERRORS
+
+enum class E(val value: String) {
+    E1(<caret>Companion.foo);
```

Comment:
This is a valid warning, `E.Companion.foo` is being referenced as `Companion.foo`, which is redundant since the `companion object` is in scope. This should be `E1(foo)` in `.after.kt`.